rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeper.java

```diff
@@ -62,6 +65,8 @@
  */
 public class BookKeeper implements com.qubole.rubix.spi.BookKeeperService.Iface
 {
+  public static final String METRIC_TOTAL_BLOCK_HITS = name(BookKeeper.class, "total-block-hits");
```

Comment:
There are many suggestions for naming conventions on the internet. This is the best I found: https://docs.signalfx.com/en/latest/best-practices/naming-conventions.html Lets follow that. In this case the metric name will be `rubix.bookkeeper.total_block_hits`. Let's debate this in the design doc.

Reply:
This is better: `rubix.bookkeeper.local_cache.count`

Reply:
Metric name & accompanying variables have been renamed